src/TableV8/styled.tsx

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) 2020 The Ontario Institute for Cancer Research. All rights reserved
+ *
+ * This program and the accompanying materials are made available under the terms of
+ * the GNU Affero General Public License v3.0. You should have received a copy of the
+ * GNU Affero General Public License along with this program.
+ *  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY
+ * EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+ * OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
+ * SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
+ * TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+ * OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
+ * IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import clsx from 'clsx';
+import { styled } from '../ThemeProvider';
+
+export const TABLE_CLASSES = {
+  TABLE: 'rt-table',
+  TBODY: 'rt-tbody',
+  TD: 'rt-td',
+  TR_GROUP: 'rt-tr-group',
+  TR: 'rt-tr',
+};
+
+const Table = (props: React.PropsWithChildren<{ className?: string }>) => (
+  <table {...props} className={clsx(TABLE_CLASSES.TABLE, props.className)} />
+);
+export const StyledTable = styled(Table)`
+  border: ${({ theme }) => `solid 1px ${theme.colors.grey_2}`};
+  flex: auto 1;
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  width: 100%;
+  border-collapse: collapse;
+  overflow: auto;
+  * {
+    box-sizing: border-box;
+  }
+`;
+
+const TableBody = (props: React.PropsWithChildren<{ className?: string }>) => (
+  <tbody {...props} className={clsx(TABLE_CLASSES.TBODY, props.className)} />
+);
+export const StyledTableBody = styled(TableBody)`
+  min-width: 200px;
+  flex: 99999 1 auto;
+  display: flex;
+  flex-direction: column;
+  overflow: auto;
+`;
+
+// TODO do we need this?
```

Comment:
The react-table "row group" is an abstract entity, which can be (if needed) expressed visually and semantically, but there's no need to create a separate markup element for them. 
While I am not certain we have a need for that in our current projects, there may be need for it in the future... bridge we can cross when we get there.

i.e. no need for a div to literally group the TRs, when they can be grouped by a `.className` and `data-` tags

PS: not a big deal, but `divs` are not valid markup in a `Table`, unless inside of `TD`s